src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -847,19 +773,56 @@ def check_error_and_log(error_regex, log_message_format):
     while self.should_try_fuzzing(max_total_time, num_retries):
       # Increment this now so that we can just "continue" without incrementing.
       num_retries += 1
-      # If we have spent a while trying to fuzz and are still retrying, then
-      # make sure we use AFL_FAST_CAL, otherwise we won't have much time for
-      # fuzzing.
-      self.strategies.decide_fast_cal_manual(get_time_spent_fuzzing())
-      if self.strategies.use_fast_cal:
-        environment.set_value(constants.FAST_CAL_ENV_VAR, 1)
 
       self.afl_setup()
+
+      # If the target was compiled for CMPLOG we need to set this.
+      build_directory = environment.get_value('BUILD_DIR')
+      if not build_directory:  # Required for passing the unit tests.
+        build_directory = '/out'
```

Comment:
the unit tests fail as build_directory is None.
does this make the unit tests wrong or this code?

Reply:
when I try to do this I trigger a different assert:
```
FAIL: test_setup (tests.core.build_management.build_manager_test.AuxiliaryRegularBuildTest)
Tests setting up a build.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/clusterfuzz/src/python/tests/core/build_management/build_manager_test.py", line 1058, in test_setup
    self._assert_env_vars()
  File "/clusterfuzz/src/python/tests/core/build_management/build_manager_test.py", line 1025, in _assert_env_vars
    self.assertIsNone(os.environ.get('BUILD_DIR'))
AssertionError: '/tmp/afl_build_dir' is not None
```